src/js/components/molecules/list-section/list-section.html

```diff
@@ -1,28 +1,31 @@
-<section class="list-section">
-    <header class="list-section__header">
-        <h3 class="list-section__title" v-if="title">{{ title }}
-            <span class="list-section__count">({{cards.length}})</span>
+<section class="list-section" :class="defineCardTypesClass()">
+    <template v-if="title">
+        <header class="list-section__header">
+            <h3 class="list-section__title">{{ title }}
+                <span class="list-section__count">({{cards.length}})</span>
+                <template v-if="tooltip">
```

Comment:
Technically the tooltip should be positioned outside the header. That would prevent cascading styles

Reply:
Tooltip was already positioned in the header. I would suggest to pick this up seperately, looking at the size of this work already